homeassistant/components/bang_olufsen/media_player.py

```diff
@@ -393,7 +498,10 @@ def media_duration(self) -> int | None:
     @property
     def media_position(self) -> int | None:
         """Return the current playback progress."""
-        return self._playback_progress.progress
+        # Don't show progress if the the device is a Beolink listener.
```

Comment:
Why? In the Sonos integration which supports group all the media players in the group show the playback progress.

Reply:
If I remember correctly, the progress is ever-increasing (for listeners) for the duration of the multi-room session, which is not particularly useful. I'll check in the morning.

Reply:
In Sonos all the non-leader media players report the status from the leader and also Route the transport commands to the leader.  In that pausing any speaker in the group cause all to pause which for that integration seems like a reasonable solution. Whether that's the right choice here, I don't know.

Reply:
That's how Chrome Cast groups work in Home Assistant too; status is read from the group leader and any action is forwarded to the group leader.

Reply:
Looks like the playback progress works as expected for listeners now, so I'll remove this check. 

For Beolink multi-room experiences, the listeners will connect to a leader and simply play the "Beolink" source that the leader provides (and get all the relevant metadata), this means that pausing listeners will only pause that specific listener, while pausing the leader will pause the playback on the leader, which will in turn "pause" playback on the listeners.